en_US.json

```diff
@@ -310,6 +310,7 @@
     "FORCE_SHOW_HUD_ELEMENT": "Force show HUD element",
     "FORCE_SHOW_HUD": "Force show HUD",
     "FORCE_WANTED_LVL": "Force Wanted Level",
+    "FORCE_WANTED_LVL_INFO": "Force Others or Yourself into a specific Wanted Level",
```

Comment:
I'd change this text to `Keep a specific wanted level active on yourself.`

Reply:
I suppose you are right xD